### ISSUE TITLE
cls not defined in import_ownertrust method

### ISSUE THREAD

**Issue by antevens** [isislovecruft/python-gnupg#141]
I think either the method should use the instance or it should be a class method and take cls as the first argument or be `self.__cls__`?

```
def import_ownertrust(self, trustdb=None):
    """Import ownertrust from a trustdb file.
    :param str trustdb: The path to the trustdb.gpg file. If not given,
                        defaults to :file:`trustdb.gpg` in the current GnuPG
                        homedir.
    """
    if trustdb is None:
        trustdb = os.path.join(cls.homedir, 'trustdb.gpg')

    import_proc = cls._open_subprocess(['--import-ownertrust'])
    tdb = open(trustdb, 'rb')
    _util._threaded_copy_data(tdb, import_proc.stdin)
```

Could also be replaced with something nicer, for example: (untested)

```
     def import_ownertrust(self, trustdb=None, trusts=None):
        """
        Accepts either a trustdb file or a dictionary of fingerprint:trust_level
        """
        from gnupg._util import _make_binary_stream
        result = None
        if trustdb is None and trusts is None:
            trustdb = os.path.join(self.homedir, 'trustdb.gpg')

        try:
            tdb = open(trustdb, 'rb')
        except (IOError, TypeError):
            print('IOERROR')
            if trusts is not None:
                trust_string = ''
                for fingerprint, level in trusts.iteritems():
                    trust_string += ('{0}:{1}:\n'.format(fingerprint, level))
                tdb = _make_binary_stream(trust_string, self._encoding)
                print('Made stream')
            else:
                raise

        if self.is_gpg2():
            import_proc = self._open_subprocess(['--import-ownertrust'])
            gnupg._util._threaded_copy_data(tdb, import_proc.stdin)
            print('Copying')
        else:
            result = pseudo_result()
            args = ["--import-ownertrust", trustdb]
            self._handle_io(args, tdb, result, binary=False)

        tdb.close()
        return import_proc or result
```

**Comment by isislovecruft** [comment]
Hi @antevens! Thanks for reporting this! You're correct that it should be `cls`, not `self`. I'm going to go ahead and fix that now, but if you'd like to open a PR for adding `trusts=` as a string, I'd be happy to take the patch.

**Comment by isislovecruft** [comment]
Fixed in python-gnupg version 2.2.0.
